demos/chips.html

```diff
@@ -53,6 +53,27 @@
         </div>
       </section>
 
+      <section class="example">
+        <h2>Choice Chips</h2>
+        <div class="mdc-chip-set mdc-chip-set--choice">
+          <div class="demo-chip mdc-chip">
```

Comment:
These chips should probably have `tabindex="0"` and one or more [ARIA attributes](https://www.w3.org/TR/wai-aria-1.1/#widget_roles).

I'm guessing you'll need something like this (but definitely double check first, cuz I'm not an ARIA expert):

```html
<div class="mdc-chip-set mdc-chip-set--choice" role="radiogroup">
  <div class="demo-chip mdc-chip" role="radio" aria-checked="false">
    <div class="mdc-chip__text">Extra Small</div>
  </div>
  ...
</div>
```

The JS will need to update the `aria-checked` attribute whenever a chip is selected/activated (I forget what the difference is between the two).

You'll also need to add styles for the `:focus` state.

Reply:
FYI I left a similar comment related to what elements we should use and which elements should be focusable on #2191.

Reply:
Ah yes, thanks for bring this up! Let's handle it in #2191.